src/Raven.Studio/wwwroot/App/views/database/settings/customAnalyzers.html

```diff
@@ -1,15 +1,15 @@
-<div class="flex-vertical absolute-fill content-margin">
+<div class="flex-vertical absolute-fill content-margin custom-analyzers">
     <div class="row flex-row flex-grow flex-stretch-items">
         <div class="col-sm-12 col-lg-6 flex-vertical">
             <div class="flex-header">
                 <div>
                     <a class="btn btn-primary" data-bind="attr: { href: addUrl }">
-                        <i class="icon-plus"></i><span>Add Custom Analyzer</span>
+                        <i class="icon-plus"></i><span>Add a Database Custom Analyzer</span>
```

Comment:
I wonder if it is good to add word database here. To avoid meaning that this is Database Custom Analyzer. Maybe let's roll it back to Custom Analyzers and also have: Server-Wide Custom Analyzers?